### ISSUE TITLE
paprefs doesn't start in KDE5 15.09

### ISSUE THREAD

**Issue by CMCDragonkai** [NixOS/nixpkgs#13610]
Trying to launch paprefs results in this error, and the program aborts.

```
> paprefs

GConf Error: Failed to contact configuration server; the most common cause is a missing or misconfigured D-Bus session bus daemon. See http://projects.gnome.org/gconf/ for information. (Details -  1: GetIOR failed: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.GConf was not provided by any .service files)
terminate called after throwing an instance of 'Gnome::Conf::Error'
```

@abbradar

**Comment by abbradar** [comment]
Please try this patch: https://github.com/abbradar/nixpkgs/commit/eb57a40b30be0ff5b2cce6d23dc804c44dd5c5dc

**Comment by abbradar** [comment]
@CMCDragonkai ping?

**Comment by CMCDragonkai** [comment]
@abbradar It works. No more errors. I get a UI pop up called PulseAudio Preferences. It looks kind of strange, but that's probably due to something else.

**Comment by abbradar** [comment]
Pushed fix to master and 16.03. Thanks!
